### DISCUSSION TITLE
'nativeAutomation' is deprecated and shouldn't be used.

### DISCUSSION THREAD

**Discussion by ghost** [leancodepl/patrol#1645] [General]
In this case, I guess the default value of the "ativeAutomation" should be "true", not "false".
<img width="438" alt="image" src="https://github.com/leancodepl/patrol/assets/108880869/50dac928-1d4a-4b71-acb2-fd100400810b">

**Comment by jBorkowska** [comment]
Actually the deprecation message is misleading, we will fix it soon.

We don't want to change the default, because it will cause a breaking change. This parameter will be removed in the future and `nativeAutomation` will always be set to `true`
